www/annotationdb/annotationdb.css

```diff
@@ -0,0 +1,443 @@
+div.wy-nav-content{
```

Comment:
It looks like the left margin is not in line with other docs pages. Preview is here: https://ci.hail.is/repository/download/HailSourceCode_HailMainline_BuildDocs/29108:id/www/index.html